CHANGELOG

```diff
@@ -1,4 +1,17 @@
-v 1.1.0 (Next)
+v 1.3.0 (20 Sep 2018)
+- Bump default Jeknins to 2.121
```

Comment:
Typo, should be "Jankies", not "Jeknins"

Reply:
Yes, changed to "Jankies".